R/barchart.R

```diff
@@ -162,7 +163,7 @@ Bar <- function(x,
         is.stacked <- FALSE
     }
     is.hundred.percent.stacked <- grepl("100% Stacked", type, fixed=T)
-    if (any(is.na(as.matrix(chart.matrix))))
+    if (any(!is.finite(as.matrix(chart.matrix))))
         warning("Missing values have been set to zero.")
```

Comment:
Should the warning be updated to capture the case of `Inf` that this change will now capture?

Reply:
I thought about it, but changing the warning here causes a lot of tests to fail. In column chart, this warning is also used even though it has always used `is.finite` instead of `is.na`. I think the reason is that many users would have missing values in their data, whereas if they  saw a warning about infinite values, they would probably be wondering how to put infinite values in.